mappings/net/minecraft/client/gui/screen/ingame/BookEditScreen.mapping

```diff
@@ -38,6 +39,30 @@ CLASS net/minecraft/class_473 net/minecraft/client/gui/screen/ingame/BookEditScr
 		ARG 1 keyCode
 		ARG 2 scanCode
 		ARG 3 modifiers
-	CLASS class_475 Position
+	METHOD method_27576 getRenderCache ()Lnet/minecraft/class_473$class_5233;
+	METHOD method_27577 resetCache ()V
+	METHOD method_27578 createRenderCache ()Lnet/minecraft/class_473$class_5233;
+	CLASS class_475 Line
+		FIELD field_24278 style Lnet/minecraft/class_2583;
+		FIELD field_24279 content Ljava/lang/String;
+		FIELD field_24280 text Lnet/minecraft/class_2561;
 		FIELD field_2853 y I
 		FIELD field_2854 x I
+		METHOD <init> (Lnet/minecraft/class_2583;Ljava/lang/String;II)V
+			ARG 2 content
+			ARG 3 x
+			ARG 4 y
+	CLASS class_5233 RenderCache
```

Comment:
is it really a cache, or it is just the current content, as this is an editing screen it is quite weird to have this name

Reply:
PageContent? might be a better name? I struggled with this one for a bit.

Reply:
or `PageContent`? should mention page if this is page specific.

Reply:
PageContent should work

Reply:
Now uses PageContent